CMakeLists.txt

```diff
@@ -344,6 +344,7 @@ if (NOT STB_IMAGE_FOUND)
         install(FILES ${CMAKE_BINARY_DIR}/deps/${STB_DIR}/stb_image.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
         install(FILES ${CMAKE_BINARY_DIR}/deps/${STB_DIR}/stb_image_write.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
         add_definitions(-DHAS_STB)
+        set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${CMAKE_BINARY_DIR}/deps/${STB_DIR}/)
```

Comment:
I think this is a duplication of line 339.

Reply:
You're right, thanks. :+1: